app/views/layouts/angular-bootstrap/_endpoints_angular.html.haml

```diff
@@ -77,6 +77,16 @@
                           "checkchange"             => "",
                           "prefix"                  => prefix.to_s,
                           "reset-validation-status" => "#{prefix}_auth_status"}
+      - detect = "detectClicked({target: '.detect_button'}, true)"
```

Comment:
The indent level should be the same as that `%span` on the next line, otherwise, it looks like you're declaring a variable for a block that is no longer there.